docs/user/reporting/network-policy.asciidoc

```diff
@@ -0,0 +1,74 @@
+[role="xpack"]
+[[reporting-network-policy]]
+=== Restricting requests with a Reporting Network Policy
+
+When {report-features} generates PDF reports, it uses the Chromium browser to fully load the Kibana page on the server. This
+potentially involves sending requests to external hosts, for example, a request may go to an external image server for showing a
+field formatted as an image, or to show an image in a Markdown visualization.
+
+If the Chromium browser is requested to send a request that violates the network policy, Reporting will stop processing the page
+before the request goes out, and the report will be marked as a failure. Additional information about the event can be found in
+Kibana's server logs.
+
+A network policy applies not only to outgoing requests, but also incoming responses. That means if a request goes out to an allowed
+host, but is redirected and a response returns from a denied host, the response will be denied, and the report will fail.
+
+[NOTE]
+============
+Kibana installations are not designed to be publicly accessible over the Internet. The Reporting network policy and other capabilities
```

Comment:
```suggestion
{kib} installations are not designed to be publicly accessible over the Internet. The Reporting network policy and other capabilities
```

Is this true? ESS is publicly accessible, you just need our security features enabled to protect the installation